docs/source/developers/python.rst

```diff
@@ -518,13 +518,11 @@ We set a number of environment variables:
 
 - the path of the installation directory of the Arrow C++ libraries as
   ``ARROW_HOME``
-- add the path of installed DLL libraries to ``PATH``
 - and the CMake generator to be used as ``PYARROW_CMAKE_GENERATOR``
 
 .. code-block::
 
-   $ set ARROW_HOME=%cd%\arrow-dist
-   $ set PATH=%ARROW_HOME%\bin;%PATH%
+   $ set ARROW_HOME=%CONDA_PREFIX%
    $ set PYARROW_CMAKE_GENERATOR=Visual Studio 15 2017 Win64
```

Comment:
This could also be "Ninja"? (based on the appveyor script) 

But I am not familiar enough with Windows to know if there is a good reason to prefer one or the other (on Linux we recommend to use Ninja)

Reply:
Locally I have built with `Ninja` but I also do not know what is preferred on Windows.

Reply:
The C++ docs mention both with some + if using `Ninja`:
https://arrow.apache.org/docs/dev/developers/cpp/windows.html#building-using-visual-studio-msvc-solution-files

Reply:
If the C++ docs suggest ninja, I think we can be opinionated here in the Python section and also use Ninja